### PR TITLE
Fix #3819 & #3821 - Minor JSON Schema fixes

### LINKED ISSUE #3819: Review boolean defaults in ingestionPipeline and airflowConfig
After merging https://github.com/open-metadata/OpenMetadata/pull/3804 in, let's review the defaults of bool types of the introduced JSON schemas.

Some files have default as a string which will impact the REST usage.

### PR DESCRIPTION
**Pull request by pmbrull**
### Describe your changes :

This PR fixes https://github.com/open-metadata/OpenMetadata/issues/3821 and fixes https://github.com/open-metadata/OpenMetadata/issues/3819
#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Review by harshach — APPROVED** [review]

File: catalog-rest-service/src/main/resources/json/schema/type/filterPattern.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "title": "MetadataWorkflow",
+  "title": "FilterPattern",
   "description": "OpenMetadata Ingestion Framework definition.",
   "type": "object",
   "definitions": {
     "filterPattern": {
       "type": "object",
-      "javaType": "org.openmetadata.catalog.metadataingestion.FilterPattern",
+      "javaType": "org.openmetadata.catalog.metadataIngestion.FilterPattern",
```
Comment (pmbrull) [inline catalog-rest-service/src/main/resources/json/schema/type/filterPattern.json:10]: Updated naming

File: catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/services/DatabaseServiceResourceTest.java [hunk 2]
```diff
@@ -36,8 +36,8 @@
 import org.openmetadata.catalog.entity.services.ingestionPipelines.IngestionPipeline;
 import org.openmetadata.catalog.jdbi3.DatabaseServiceRepository.DatabaseServiceEntityInterface;
 import org.openmetadata.catalog.metadataIngestion.DatabaseServiceMetadataPipeline;
-import org.openmetadata.catalog.metadataingestion.FilterPattern;
-import org.openmetadata.catalog.metadataingestion.SourceConfig;
+import org.openmetadata.catalog.metadataIngestion.FilterPattern;
+import org.openmetadata.catalog.metadataIngestion.SourceConfig;
```
Comment (pmbrull) [inline catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/services/DatabaseServiceResourceTest.java:40]: Updated imports

File: catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/services/ingestionpipelines/IngestionPipelineResourceTest.java [hunk 3]
```diff
@@ -370,7 +370,7 @@ void put_IngestionPipelineForDashboardSourceUpdate_200(TestInfo test) throws IOE
                         .withStartDate(startDate.toString())),
             OK,
             ADMIN_AUTH_HEADERS);
-    String expectedFQN = EntityUtil.getFQN(SUPERSET_REFERENCE.getName(), ingestion.getName());
+    String expectedFQN = FullyQualifiedName.build(SUPERSET_REFERENCE.getName(), ingestion.getName());
```
Comment (pmbrull) [inline catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/services/ingestionpipelines/IngestionPipelineResourceTest.java:373]: Using the new FQN

**Comment by pmbrull** [comment]
Opened https://github.com/open-metadata/OpenMetadata/issues/3875 to check sonarcloud bug

**Pull request merged**